src/getWorkout.js

```diff
@@ -1,32 +1,17 @@
 const express = require('express');
 const port = 5000;
-const http = require('http');
 const fs = require('fs');
 const app = express();
 
-const httpServer = http.createServer((req, res) => {
+app.get('/', (req, res) => {
     fs.readFile('../data/workout_list', 'utf-8', (err, workoutList) => {
-        if(!err){
-            res.writeHead(200, {'Access-Control-Allow-Origin':'*'});
-            res.end(JSON.stringify(workoutList));
-        } else {
-            res.writeHead(404, {'Access-Control-Allow-Origin':'*'});
-            res.end(JSON.stringify(err));
-        }
+        res.append('Access-Control-Allow-Origin', ['*']);
+        res.append('Access-Control-Allow-Methods', 'GET,PUT,POST,DELETE');
```

Comment:
Are we allowing all the methods on current url location?

Think of `Access-Control-Allow-Methods` as a navigator to users. This means the user can call this service with `/` location by `get, put, post, delete`. However, there is only `get` method here. So in this case, I think you should only allow users to use `get` method here.

Reply:
Changed.